src/public/locales/en/default_translation_en.json

```diff
@@ -6,16 +6,21 @@
   "Portal": "Portal",
   "missing": "missing",
   "entities": {
-    "Phenopackets": "Clinical Data",
+    "phenopacket_one": "Phenopacket",
+    "phenopacket_other": "Clinical Data",
+    "individual_one": "Individual",
+    "individual_other": "Individuals",
     "Individual": "Individual",
-    "Individuals": "Individuals",
```

Comment:
Why not `individual_one` and `individual_other` like the other entities? Mismatches with the FR translation

Reply:
oops - forgot